src/NHibernate/Cfg/XmlHbmBinding/PropertiesBinder.cs

```diff
@@ -402,30 +386,27 @@ private void BindCollectionProperty(ICollectionPropertiesMapping collectionMappi
 			property.Cascade = collectionMapping.Cascade ?? mappings.DefaultCascade;
 		}
 
-		private Property CreateProperty(IEntityPropertyMapping propertyMapping, string propertyOwnerClassName, IValue value, IDictionary<string, MetaAttribute> inheritedMetas)
```

Comment:
In all cases the caller was passing `className` as `propertyOwnerClassName` argument.